README.md

```diff
@@ -118,26 +118,26 @@ equality. See the "Equality classes" section for more information.
 
 `Data.Singletons.TH` (from `singletons-th`) exports all the definitions needed
 to use the Template Haskell code to generate new singletons.
-`Data.Singletons.Prelude.TH` (from `singletons-bsae`) re-exports
+`Data.Singletons.Base.TH` (from `singletons-bsae`) re-exports
```

Comment:
```suggestion
`Data.Singletons.Base.TH` (from `singletons-base`) re-exports
```

Reply:
Eep, good catch. Thankfully, this appears to be the only occurrence of this typo (according to `grep`).